pycaption/base.py

```diff
@@ -217,6 +240,15 @@ def _format_timestamp(self, value, msec_separator=None):
         return u'0' + str_value
 
 
+class CaptionList(list):
```

Comment:
At the moment the specialized_collections module is SCC specific, so I will leave it in the base module for now.